### DISCUSSION TITLE
Are the Trinity v1.2 better or not?

### DISCUSSION THREAD

**Discussion by rowild** [janhq/jan#1344] [Get Help]
This might be an incredibly stupid question, but I am being very honest when I say that I am truly very confused about all this AI things. Therefore I kindly ask for your patience with my stupidity and help me get smarter, when I humbly ask, whether the Trinity v1.2 gguf model is better than the v1, which is standard when downloaded via the app? 

I would expect that the latest version of an app also downloads the latest version of a model, but I know that this thinking is not always correct.

Any clarification would be most welcome! Thank you!

**Comment by Brog33** [comment]
Just thought to add that Trinity 1.2 tend to gives me more accurate answers than Mistral 7B when asking random questions about stuff.
